pkg/apis/pipeline/v1/pipelinerun_defaults_test.go

```diff
@@ -37,13 +38,42 @@ func TestPipelineRunSpec_SetDefaults(t *testing.T) {
 		prs  *v1.PipelineRunSpec
 		want *v1.PipelineRunSpec
 	}{
+		{
+			desc: "timeouts is nil",
+			prs:  &v1.PipelineRunSpec{},
+			want: &v1.PipelineRunSpec{
+				TaskRunTemplate: v1.PipelineTaskRunTemplate{
+					ServiceAccountName: config.DefaultServiceAccountValue,
+				},
+				Timeouts: &v1.TimeoutFields{
+					Pipeline: &metav1.Duration{Duration: config.DefaultTimeoutMinutes * time.Minute},
+				},
+			},
+		},
+		{
+			desc: "timeouts is not nil",
```

Comment:
can you add a test case with an existing `timeouts.pipeline`?

Reply:
thanks added a case where `timeouts.pipeline` value is not default so to check it is not set as Default.